Goobi/config/goobi_config.properties

```diff
@@ -115,20 +115,24 @@ useOrigFolder=true
 # to have sufficient permissions to do whatever they should do.
 #
 # For examples of these scripts, see:
-# https://github.com/kitodo/kitodo-production/tree/1.11.x/Goobi/scripts
+#
+# https://github.com/kitodo/kitodo-production/tree/2.0.0/Goobi/scripts
+#
+# for script files of release 2.0.0. Never versions could be found under
+# appropriate release tag.
```

Comment:
s/Never/Newer/. The link currently does not work, because there is no 2.0.0 tree.

For now, I suggest this comment:

    # For examples of these scripts, see:
    # https://github.com/kitodo/kitodo-production/tree/master/Goobi/scripts
    # Replace "master" by your version of Kitodo to see examples matching that version.

For the future, we should consider adding those scripts to the distribution and use those default scripts if there is no special configuration. Are there installations which override the scripts? If not, then why not remove that nuisance of requiring an installation in /usr/local?